deploy/cluster-operator/templates/deployment.yaml

```diff
@@ -21,7 +21,14 @@ spec:
       containers:
         - name: {{ .Chart.Name }}
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag }}"
-          imagePullPolicy: Always
+          imagePullPolicy: Never
```

Comment:
Always is more secure, in case a bad actor messes with docker daemon. It's useful to put imagePullPolicy in values.yaml to avoid these comments :p

Reply:
Your right, I had that in so I could feed an image to Kind so I wasnt uploading a bunch of bad tags to dockerhub and forgot to revert. Let me fix that.

Reply:
It's alright, Docker isn't charging us per image storage. It's going to be easier long term to ensure you're testing the code that was just written. Doing a bunch of testing on a cached image with old code is a terrible day.

Later, we could build some optimizations around kind in the makefile. `kind load` doesn't depend on pushing images up to docker hub